packages/core/src/validations.ts

```diff
@@ -40,18 +39,21 @@ export const validateZen = (singleContext: any, block: string) => {
   }
 }
 
+const excludeProps = ["zenContent"]
+
 /**
  * Function responsible to check if paths in the yml containing same folder
  * @param {ymlContent} object yml object
  */
-export const validatePathsInYml = (ymlContent: any) => {
+export const validatePathsInYml = (ymlContent: any, user: string | null) => {
```

Comment:
Restroom does not have concept of what a user is... if it's a new entity, then the documentation is lacking